amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2762,6 +2807,28 @@ void fillAudioDescriptionPtrFromDims(RpptDescPtr &descPtr, size_t *maxTensorDims
     }
 }
 
```

Comment:
please combine all 3 functions into 1: Use layout to differentiate

Reply:
Will issue this as a separate PR as this would require changes in all the Audio kernel files